package.json

```diff
@@ -47,8 +44,11 @@
     "chokidar-cli": "^1.2.0",
     "live-server": "^0.9.2",
     "mkdirp": "^0.5.1",
+    "mofo-bootstrap": "^1.0.1",
     "node-sass": "^3.4.2",
     "npm-run-all": "^1.7.0",
+    "react": "^15.0.0",
```

Comment:
I believe that the consumer will only install regular deps for its libraries. If it doesn't have React as a dep, it will warn that it's missing a peer.

For learning, React 14 is installed.

On the mofo-ui side, it will install 15 for itself.